csm/plugins/cortx/rgw.py

```diff
@@ -149,8 +149,14 @@ def _create_error(self, status: int, body: dict) -> Any:
         Log.error(f"Create error body: {body}")
 
         rgw_error = RgwError()
-        rgw_error.http_status = status
-        rgw_error.error_code = RgwErrors[body['Code']]
-        rgw_error.error_message = rgw_error.error_code.value
-
+        try:
+            rgw_error.http_status = status
+            rgw_error.error_code = RgwErrors[body['Code']]
+            rgw_error.error_message = rgw_error.error_code.value
+        except KeyError:
```

Comment:
What if other than KeyError is raised due to any unknown issue? Should we handle base exception too?

Reply:
The only possible error at this stage in the code flow is 'KeyError'.